resources/startstopserver.ts

```diff
@@ -20,63 +20,64 @@ import { ECSClient, UpdateServiceCommand } from '@aws-sdk/client-ecs';
-    }
-    
-    if (event.queryStringParameters && event.queryStringParameters.desiredCount !== undefined) {
-        let count = Math.min(Math.max(+event.queryStringParameters.desiredCount, 0), 1);
-        params.desiredCount = count;
-        console.log("changing desiredCount to " + count);
-    }
+  const params = {
+    desiredCount: 1,
+    service: SERVICE_NAME,
+    cluster: CLUSTER_ARN
+  }
 
-    if (event.queryStringParameters && event.queryStringParameters.key) {
-        let key = event.queryStringParameters.key;
-        if (key == PASSWORD) {
-            const client = new ECSClient({ region: REGION });
-            console.log("starting service " + JSON.stringify(params));
-            message = "authentication success";
-            responseCode = 200;
+  if (event.queryStringParameters && event.queryStringParameters.desiredCount !== undefined) {
+    let count = Math.min(Math.max(+event.queryStringParameters.desiredCount, 0), 1);
+    params.desiredCount = count;
+    console.log("changing desiredCount to " + count);
+  }
 
+  if (event.queryStringParameters && event.queryStringParameters.key) {
+    let key = event.queryStringParameters.key;
+    if (key == PASSWORD) {
+      const client = new ECSClient({region: REGION});
+      console.log("starting service " + JSON.stringify(params));
+      message = "authentication success";
+      responseCode = 200;
 
-            const updateCommand = new UpdateServiceCommand(params);
 
-            client.send(updateCommand).then(
-                (data) => {console.log(data);},
-                (err) => {    console.log(err);}
-            );
+      const updateCommand = new UpdateServiceCommand(params);
+
+      await client.send(updateCommand).then(
+        (data) => {
+          console.log(`OK! ${JSON.stringify(data, null, 2)}`);
+        },
+        (err) => {
+          console.log(`Error! ${JSON.stringify(err, null, 2)}`);
         }
+      );
     }
+  }
 
-    let responseBody = {
-        message: message,
-    };
-    
-    // The output from a Lambda proxy integration must be 
-    // in the following JSON object. The 'headers' property 
-    // is for custom response headers in addition to standard 
-    // ones. The 'body' property  must be a JSON string. For 
-    // base64-encoded payload, you must also set the 'isBase64Encoded'
-    // property to 'true'.
-    return {
-        statusCode: responseCode,
-        headers: {
-        },
-        body: JSON.stringify(responseBody)
-    };
-};
+  let responseBody = {
+    message: message,
+  };
 
-export default handler;
+  // The output from a Lambda proxy integration must be
+  // in the following JSON object. The 'headers' property
+  // is for custom response headers in addition to standard
+  // ones. The 'body' property  must be a JSON string. For
+  // base64-encoded payload, you must also set the 'isBase64Encoded'
+  // property to 'true'.
+  return {
+    statusCode: responseCode,
+    headers: {},
```

Comment:
Not needed, and the comment above is redundant if the handler response is strongly typed